### PR TITLE
feat(sidebar): open up broadcasts if #whats-new is the hash

### PR DESCRIPTION
**Pull request by scefali**
We want to be able to send links to Sentry with the "What's new" section opened. We can do this with a hash parameter `#sidebar-broadcasts`.

### CODE REVIEW

File: static/app/components/sidebar/index.tsx [hunk 1]
```diff
@@ -91,6 +91,12 @@ function Sidebar({location, organization}: Props) {
     return () => bcl.remove('body-sidebar');
   }, [bcl]);
 
+  useEffect(() => {
+    if (location?.hash === '#whats-new') {
+      togglePanel(SidebarPanelKey.Broadcasts);
+    }
+  }, [location?.hash]);
```
Comment (evanpurkhiser) [inline static/app/components/sidebar/index.tsx:98]: Want to generalize it be 
```tsx
type SidebarHash = `#sidebar-${SidebarPanelKey}`;
```

and then you can parse the string and toggle whatever with `#sidebar-whats-new`

Comment (scefali) [reply static/app/components/sidebar/index.tsx:98]: @evanpurkhiser good call

File: static/app/components/sidebar/index.tsx [hunk 2]
```diff
@@ -91,6 +91,19 @@ function Sidebar({location, organization}: Props) {
     return () => bcl.remove('body-sidebar');
   }, [bcl]);
 
+  useEffect(() => {
+    [
+      SidebarPanelKey.Broadcasts,
+      SidebarPanelKey.OnboardingWizard,
+      SidebarPanelKey.PerformanceOnboarding,
+      SidebarPanelKey.ServiceIncidents,
+    ].forEach(sidebarValue => {
```
Comment (evanpurkhiser) [inline static/app/components/sidebar/index.tsx:100]: can't you just do 
```suggestion
    Object.values(SidebarPanelKey).forEach(sidebarkey => {
```

Comment (scefali) [reply static/app/components/sidebar/index.tsx:100]: @evanpurkhiser oh yea good call

File: static/app/components/sidebar/index.tsx [hunk 3]
```diff
@@ -91,6 +91,14 @@ function Sidebar({location, organization}: Props) {
     return () => bcl.remove('body-sidebar');
   }, [bcl]);
 
+  useEffect(() => {
+    Object.values(SidebarPanelKey).forEach(sidebarValue => {
+      if (location?.hash === `#sidebar-${sidebarValue}`) {
```
Comment (evanpurkhiser) [inline static/app/components/sidebar/index.tsx:96]:
```suggestion
    Object.values(SidebarPanelKey).forEach(key => {
      if (location?.hash === `#sidebar-${key}`) {
```

🤷 what is 'value' lol

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**